src/main/java/benben/BenBen.java

```diff
@@ -0,0 +1,279 @@
+package benben;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * The type BenBen is a chatbot that interacts with the user by taking input from the system
+ * and prints its response as system output.
+ * It can create and edit a task list for a user.
+ */
+public class BenBen {
+
+    private final Storage storage;
+    private TaskList tasks;
+    private final Ui ui;
+
+    /**
+     * Instantiates a new BenBen with the given task list
+     *
+     * @param filePath the file path of the task list
+     */
+    public BenBen(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        try {
+            tasks = new TaskList(storage.load());
+        } catch (BenBenException e) {
+            tasks = new TaskList();
+        }
+    }
+
+    /**
+     * Creates a Todo task in the list with the specific description
+     *
+     * @param str the string command from the user
+     * @throws BenBenException when  the command from the user is of the wrong format
+     */
+    public String todo(String str) throws BenBenException{
```

Comment:
Just wondering, but should these actions be handled by the BenBen class or some other relevant classes? it may be worthwhile to explore abstracting these out into another class